packages/react-composites/tests/browser/call/HorizontalGallery.test.ts

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { test } from './fixture';
+import { expect } from '@playwright/test';
+import { buildUrlWithMockAdapter } from './utils';
+import { dataUiId, pageClick, waitForPageFontsLoaded, waitForSelector } from '../common/utils';
+import { TestRemoteParticipant } from './TestCallingState';
+import { IDS } from '../common/constants';
+
+const defaultTestRemoteParticipants: TestRemoteParticipant[] = [
+  {
+    displayName: 'Paul Bridges',
+    isSpeaking: true,
+    isVideoStreamAvailable: true
+  },
+  {
+    displayName: 'Eryka Klein'
+  },
+  {
+    displayName: 'Fiona Harper',
+    isVideoStreamAvailable: true
+  }
+];
+
+test.describe('HorizontalGallery tests', async () => {
+  test('HorizontalGallery should have 1 audio participant', async ({ pages, serverUrl }) => {
+    const page = pages[0];
+    await page.goto(
+      buildUrlWithMockAdapter(serverUrl, {
+        remoteParticipants: defaultTestRemoteParticipants
+      })
+    );
+    // Click off the screen to turn away initial aria label
+    await page.mouse.click(-1, -1);
+    await waitForPageFontsLoaded(page);
```

Comment:
instead of waiting for fonts to have loaded, is there a page element that could be waited for instead waitForPageFontsLoaded won't have guaranteed that react has done all the necessary render passes to have fully rendered the app

Reply:
I tried waiting for the VideoGallery instead but often fails because the text has not yet loaded as seen in the actual snapshot:
![horizontal-gallery-actual](https://user-images.githubusercontent.com/79475487/151271336-24f38a41-b13e-4e3d-a271-11cab8c5b524.png)
I tried tried using your waitForPageFontsLoaded and it always passes so I figure it was the bottleneck. I could also wait for both if that is better.

Reply:
Interesting there is *no* font and not just an incorrect one (i.e. that segoe ui hadn't loaded yet so fallsback to arial or something). Yeah put in a wait for both these things then